apps/www/middleware.ts

```diff
@@ -1,5 +1,5 @@
 import { NextResponse } from "next/server";
-import { authMiddleware, clerkClient } from "@clerk/nextjs";
+import { authMiddleware } from "@clerk/nextjs";
```

Comment:
Looks OK, but clerkclient is going to be used for workspaces. Take it inn again, and ill merge it. Thanks ✨